task_1/dmitriy.osipov/src/main/java/com/task/utils/DataUtility.java

```diff
@@ -0,0 +1,43 @@
+package com.task.utils;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
+public class DataUtility {
+
+  public static String getHash(Object object) {
+    String hash = null;
+    try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
```

Comment:
Oh, I see. I need to move ObjectWriter initialization after the ByteArrayOutputStream one